Creational/FactoryMethod/Bicycle.php

```diff
@@ -1,9 +1,11 @@
 <?php
+declare(strict_types = 1);
 
 namespace DesignPatterns\Creational\FactoryMethod;
 
 /**
- * Bicycle is a bicycle
+ * Class Bicycle
```

Comment:
This type of comments is non sens. Why do you have to write `Class Bicycle` when is clearly that you define this class 3 lines below ?